ui/narrative/methods/import_diff_expression/display.yaml

```diff
@@ -0,0 +1,83 @@
+name     : Import Differential Expression Matrix from CSV/Excel File in Staging Area
+subtitle : Import a CSV or Excel file from your staging area into your Narrative as an DifferentialExpressionMatrix data object
+tooltip  : Import a CSV or Excel file from your staging area into your Narrative as an DifferentialExpressionMatrix data object
```

Comment:
Maybe just "Import Differential Expression Matrix from Staging Area" for the title and "Import a CSV, TSV or Excel file from your staging area into your Narrative as an DifferentialExpressionMatrix data object"